### PR TITLE
Add v2 to the DefaultBaseUrl and remove v2 in every request url string.

### PR DESCRIPTION
**Pull request by jchen293**
Add '**v2**' to the DeefaultBaseUrl and remove '**v2**' in every request URL string.

### CODE REVIEW

**Comment by Justintime50** [comment]
This looks good, wanna wait till tomorrow maybe so we can have the CI workflow run all the tests against these changes.

**Comment by nwithan8** [comment]
> This looks good, wanna wait till tomorrow maybe so we can have the CI workflow run all the tests against these changes.

This works locally

**Comment by Justintime50** [comment]
@jchen293 please rebase this branch before I merge it.

File: CHANGELOG.md
Comment (Justintime50) [inline CHANGELOG.md:7]: Nit: This reads a bit nicer. Also, please add a newline after the bullet point so we properly separate each section (the bullet point should have a newline before the next header section)

```suggestion
* Update the `DefaultApiBase` to include `v2` and remove `v2` from every request url string
```

**Review by Justintime50 — APPROVED** [review]
This looks good, only the `v2` portions of the urls were adjusted.

**Review by nwithan8 — APPROVED** [review]

**Pull request merged**
